### ISSUE TITLE
Error code 403

### ISSUE THREAD

**Issue by jadephoria** [hyugogirubato/Kamyroll#12]
Whenever I try to sign up it gives me error code 403 and then launches crunchyroll in my browser. Am I doing something wrong?

**Comment by hyugogirubato** [comment]
No, in fact the API used in the application to create a crunchyroll account is no longer up to date and during the last update I did not have time to take care of it. So it redirects to the crunchyroll page when the app fails to create an account. So create a crunchyroll account on the website and then log into the app afterwards.
